src/components/EmailEditor/store/emailEditorConfig.js

```diff
@@ -193,9 +195,20 @@ export default {
 
 		// Elements
 		toggleElement(state, payload) {
+			if (payload.type === "content") {
```

Comment:
I think we misunderstood each other here a bit.

What I wanted is that the graphical selector should be disabled with this logic:
if the text is selected then the title and paragraph graphical selector option should have a disabled state and a false value
if the paragraph or title is selected then the graphical text selector option should have a disabled state and have a false value